app/services/container_dashboard_service.rb

```diff
@@ -34,7 +34,7 @@ def status
         :href         => get_url_to_entity(:container_node)
       },
       :containers => {
-        :count        => @ems.present? ? @ems.containers.count : Container.count,
+        :count        => @ems.present? ? @ems.containers.count : Container.where('ems_id IS NOT NULL').count,
```

Comment:
per @matthewd, `where.not(:ems_id => nil)`… or even slightly better, `where.not(:ext_management_system => nil)`